operator/src/runners/fixtures/membership.rs

```diff
@@ -66,8 +66,11 @@ pub async fn run(args: MembershipCmd) -> anyhow::Result<()> {
         .seconds
         .try_into()?;
 
+    // It is recommended that TrustingPeriod should be set as 2/3 of the UnbondingPeriod
+    let trusting_period = 2 * (unbonding_period / 3);
```

Comment:
Hm, don't you think we should allow setting of trusting period to any time less than or equal to the unbonding period? So basically adding a cli argument `--trusting-period` to genesis and maybe even fixtures cli. And if it is not provided, then use 2/3 of trusting period?

Reply:
🤦 you are of course right. Sorry, will add that xD